src/mesh/voxel_cloud.rs

```diff
@@ -1126,325 +872,70 @@ mod tests {
             NormalStrategy::Sharp,
         );
 
-        let voxel_cloud = VoxelCloud::from_mesh(&mesh, &Vector3::new(0.5, 0.5, 0.5));
+        let scalar_field = ScalarField::from_mesh(&mesh, &Vector3::new(0.5, 0.5, 0.5), 0_i16, 0);
 
-        insta::assert_json_snapshot!("sphere_after_voxelization", &voxel_cloud);
+        insta::assert_json_snapshot!("sphere_after_voxelization_into_scalar_field", &scalar_field);
     }
 
     #[test]
-    fn test_voxel_cloud_three_dimensional_to_one_dimensional_and_back_relative() {
-        let voxel_cloud = VoxelCloud::new(
+    fn test_scalar_field_three_dimensional_to_one_dimensional_and_back_relative() {
+        let scalar_field: ScalarField<i16> = ScalarField::new(
```

Comment:
Why does this test need an instance of the scalar field? Wouldn't block dimensions be enough?